src/components/ContextualMenu/ContextualMenuDropdown/ContextualMenuDropdown.tsx

```diff
@@ -140,9 +149,9 @@ const generateLink = (
       {children}
     </Button>
   );
-};
+}
 
-const ContextualMenuDropdown = ({
+function ContextualMenuDropdown<L>({
```

Comment:
I think you can say `const ContextualMenuDropdown = <L,>(...)` (note trailing comma in generic) to avoid having that parsed as JSX.